template/template.go

```diff
@@ -205,24 +205,25 @@ func (kv KV) Values() []string {
 // as this will confuse them and prevent simple things like
 // simple equality checks to fail. Map everything to float64/string.
 type Data struct {
```

Comment:
The whole point of AM is that there are only very few notifications. So I'm not worried about any network load.
Adding comment anyway.